invenio_rdm_records/theme/templates/invenio_rdm_records/details/list_entities.html

```diff
@@ -8,20 +8,20 @@
 
 {% macro list_entities(entities) %}
   {% for entity in entities %}
-    {%- if entity.identifiers and entity.identifiers[0] %}
-      {# TODO: Is the first identifier given precedence or is "Orcid" given precedence? #}
-      {# Assume first identifier is given precedence #}
-      {% set main_identifier = entity.identifiers[0] %}
-      {%- if main_identifier.scheme == "Orcid" %}
-      <a href="{{ main_identifier.identifier|pid_url('orcid') }}">
-        <img class="inline-orcid" src="{{ url_for('static', filename='images/orcid.svg') }}" />
-      </a>
-      {# TODO: Other potential main identifiers #}
-      {%- endif %}
-    {% else %}
-      {# TODO: Change for an organization icon when organizations supported #}
-      <i class="fa fa-user" aria-hidden="true"></i>
+    {# TODO: Is the first identifier given precedence or is "Orcid" given precedence? #}
+    {# Assume first identifier is given precedence #}
+    {% set main_identifier = entity.identifiers[0] if entity.identifiers and entity.identifiers[0] else {"scheme": ""} %}
+    {%- if main_identifier.scheme == "Orcid" %}
```

Comment:
idutils deliver scheme names in all lowercase, so I would stay with all lowercase scheme names. Also means we can  just call ``scheme.lower()`` to ensure that the API eats whatever you might give it (``ROR``/``ror``/``Orcid``/``ORCID``/``orcid``).

Reply:
Agreed. I am thinking of dealing with this in #51 .